express/parsers/apps/aiida/parser.py

```diff
@@ -0,0 +1,59 @@
+from express.parsers import BaseParser
+from express.parsers.apps.aiida.formats.zip import AiidaArchiveFileParseError
+from express.parsers.apps.aiida.formats.zip import AiidaZipParser
+from express.parsers.apps.aiida.formats.zip import BadAiidaArchiveZipFile
+from express.parsers.apps.aiida.formats.zip import BadZipFile
+from express.parsers.utils import lattice_basis_to_poscar
+
+from express.parsers.utils import find_fines_by_name_substring
+
+
+class AiidaArchiveParser(BaseParser):
+
+    def __init__(self, *args, **kwargs):
+        """
+        AiiDA archive (zip) file parser.
+        """
+
+        super(AiidaArchiveParser, self).__init__(*args, **kwargs)
+        self.path = self.kwargs["work_dir"]
+
+    def find_zip_files(self):
+        """
+        Find all zip-files in path.
+
+        Yields:
+            str
+        """
+
+        if self.path is not None:
+            for zip_file_path in find_fines_by_name_substring('.zip', self.path):
```

Comment:
You should merge `dev` into this branch, this typo name `find_fines_by_name_substring` has been renamed since this branch was created.

Reply:
Done.